source/components/chip.md

```diff
@@ -63,6 +63,13 @@ You can add the ability to close the chip too.
   Joe
 </q-chip>
 ```
+...And to close this
+
+<p ref="chip">
```

Comment:
Hi, Vue reference on a DOM element and not component?

Reply:
It could be component as well. It is only a simple suggestion since the documentation does not mention any

Reply:
Yes, but it is incorrect. In this case the `<p>` tag has a ref.. and it does not support the 'remove' method. Please correct the example. Ref should be on the QChip. Also please test it. Thanks!